src/modules/commander/Commander.cpp

```diff
@@ -2946,6 +2947,12 @@ Commander::run()
 			_imbalanced_propeller_check_triggered = false;
 		}
 
+
+		// No need to update here, since we updated in Commander::estimator_check,
+		// which is called earlier in Commander::run
```

Comment:
We are not sure if `update()` actually returned `true` in `Commander::estimator_check`.
Could this be a problem on the first function calls if there are no messages published yet?

Reply:
Good point, I'll make sure to init the struct properly to avoid passing random data to `set_nav_state`